### PR TITLE
Added API for mail setting

### PR DESCRIPTION
**Pull request by siddharthashw**
Fixes - ERP-API-215

**1. ProjectController.cs** - added new API GetListOfTeamMemberByProjectIdAsync
**2. IProjectRepository.cs** - added new method GetListOfTeamMemberByProjectIdAsync
**3. ProjectRepository.cs** - added new method GetListOfTeamMemberByProjectIdAsync
**4. ProjectTests.cs** - added test cases for GetListOfProjectsEnrollmentOfUserByUserIdAsync and GetListOfTeamMemberByProjectIdAsync
**5. consumerapp.component.spec.ts** - added spec for consumerapp.component.ts

### CODE REVIEW

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs [hunk 1]
```diff
@@ -313,6 +313,22 @@ public class ProjectRepository : IProjectRepository
         }
 
         /// <summary>
+        /// Method to get list of team member by project Id
+        /// </summary>
+        /// <param name="projectId">project Id</param>
+        /// <returns></returns>
```
Comment (ankit213) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: update proper xml comment

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/IProjectRepository.cs [hunk 2]
```diff
@@ -83,6 +83,13 @@ public interface IProjectRepository
...
         /// <param name="userId">user's user Id</param>
         /// <returns>list of project</returns>
-        Task<List<ProjectAc>> GetListOfProjectsEnrollmentOfUserByUserId(string userId);
+        Task<List<ProjectAc>> GetListOfProjectsEnrollmentOfUserByUserIdAsync(string userId);
+
+        /// <summary>
+        /// Method to get list of team member by project Id
+        /// </summary>
+        /// <param name="projectId">project Id</param>
+        /// <returns></returns>
```
Comment (ankit213) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/IProjectRepository.cs]: update proper xml comment

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs (hunk 1 above)
Comment (siddharthashw) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: done

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/IProjectRepository.cs (hunk 2 above)
Comment (siddharthashw) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/IProjectRepository.cs]: done

**Pull request merged**